src/nos/ports/outbound/notification_emitter.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021 - 2023 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Outbound port for sending notifications."""
+
+from abc import ABC, abstractmethod
+
+from nos.core.notifications import Notification
+
+__all__ = ["NotificationEmitterPort"]
+
+
+class NotificationEmitterPort(ABC):
+    """Emits results of a calculation."""
```

Comment:
Calculation?

Reply:
I'm not sure if this is an argument for or against copilot